dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -60,7 +60,9 @@ RUN pip install /python-wheels/{{ whl }}
 RUN python3.6 -m sonic_ax_impl install
 
 # Clean up
-RUN apt-get -y purge libpython3.6-dev curl gcc make libdpkg-perl
+RUN apt-get -y purge libpython3.6-dev libpython3.6 curl gcc make libdpkg-perl
```

Comment:
Should we move the `-y` after `purge` to align with all the other files while you're at it? (here and on line 65)?

Reply:
In my opinion, not necessary.

---
In reply to: [277117545](https://github.com/Azure/sonic-buildimage/pull/2807#discussion_r277117545) [](ancestors = 277117545)

Reply:
Sure it's not necessary, just nice to unify everything.